packages/connect/README.md

```diff
@@ -19,7 +19,17 @@ import mySubstrateChainSpec from './mySubstrateChainSpec.json';
 const myChainSpec = JSON.stringify(mySubstrateChainSpec);
 
 const scClient = createScClient();
-const chain = await scClient.addChain(myChainSpec);
+const chain = await scClient.addChain(
+  myChainSpec,
+  function jsonRpcCallback(response) {
+    console.log("response", response);
+    chain.remove();
```

Comment:
I wouldn't put this `chain.remove()`. People typically copy-paste the code in the README without trying to understand it. If someone copy-pastes this code to get started, then adds a second call to `chain.sendJsonRpc` at the very bottom, they'll struggle to understand what they did wrong.

You could add `chain.remove()` at the very bottom of the example, with a comment saying to put this at the end

Reply:
Thanks! I got rid of `chain.remove()` and I also made another set of improvements to the README.